Chapters/1-7-Chapter7.md

```diff
@@ -0,0 +1,493 @@
+
+"He would need you for longer than that to get anything out of it. And Melna could simply push the story to the news wire if she wanted to kill his leverage. What do you make of this observational study? I thought Heiko had the best observatory."
+
+"Maybe it has something to do with their space rail. Maybe they can send telescopes after all. But..."
+
+Kipnu's face froze and looked worried all of a sudden. "No... they wouldn't."
+
+"Wouldn't? Wouldn't what?" I asked, but it was too late. Fajro and the magistrate had lost patience and re-entered the room.
+
+The magistrate announced to us with his casual voice, "I'd love to stand and chat in the dark with all of you, but I've got 13 cases to cover today. Are you ready to continue?"
+
+I glanced at Kipnu, who was trying his best not to let his rage show on his face.
+
+I answered the magistrate, "I'd like to ask Fajro about the study he's conducting."
+
+"Why sure," Fajro said with a smile.
+
+"Make it quick," said the magistrate, audibly annoyed, and he stepped back to observe.
+
+"What is the goal of this project?"
+
+"Well, it's a continuation of the space rail project. I can't really talk about it without a guarantee of your cooperation. Provincial secrets."
+
+"What do you need me for?"
+
+"Like I said before, you would be a communications liaison between The Heikoi and Holragi scientists."
+
+"I'm not much of a talker, and your Holragi accent isn't thick. What do you _really_ need me for?"
+
+He sighed and looked at the lantern light against the dusty ground.
+
+"Liffa, you don't miss a thing do you. That's one of the reasons I think you'd be a good candidate for the job. The candidates we're looking for have to be lightweight, agile, observant, quick-learning, brave, and in great physical shape. They also need to be willing to take the risk."
+
+"Risk? What risk?"
+
+Our eyes met and he looked at me intently. "We need you up there, Liffa."
+
+In my flood of imagination, I finally saw the dream that he saw. The danger, the pride. The whole planet. _I could die. I could die!_ I shouted inside my head.
+
+Meanwhile, Kipnu had grown impatient. He grabbed Fajro by his tunic and got in his face.
+
+"That's a death sentence, you piece of shit."
+
+"Kipnu. Stop this," the magistrate tried to push Kipnu away, but wasn't succeeding.
+
+"Kipnu," I tried to calm him, but he didn't hear me.
+
+He pushed Fajro and held him against the wall, with Fajro letting out a cry of pain as his head batted against the stone.
+
+"You think you can prey on my family, punk?" He tightened his grip on Fajro's shirt. "Find someone else to murder. Liffa is staying here."
+
+"Kipnu. This is ridiculous. Stop this immediately." The magistrate shouted, and Kipnu backed down. Fajro collapsed to the floor with a hand behind his head, and passed out. The magistrate rushed over to check on him.
```

Comment:
Not sure if ridiculous is the right word.

Reply:
absurd